### PR TITLE
replace typing.Text with str

### LINKED ISSUE #374: replace `typing.Text` with `str` for `--py3-plus`
Perhaps another example of the same idea, replace `typing.Text` with `str` for `--py3-plus`.

Example input:

```py
if False:
    from typing import Text


def f():
    # type: () -> Text
    return "hello"
```

Example diff:

```diff
--- before.py
+++ after.py
@@ -3,5 +3,5 @@
 
 
 def f():
-    # type: () -> Text
+    # type: () -> str
     return "hello"

```

_Originally posted by @jdufresne in https://github.com/asottile/pyupgrade/issues/343#issuecomment-750930060_

### PR DESCRIPTION
**Pull request by MarcoGorelli**
closes #374

### CODE REVIEW

File: pyupgrade/_plugins/typing_text.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+from pyupgrade._ast_helpers import ast_to_offset
+from pyupgrade._data import register
+from pyupgrade._data import State
+from pyupgrade._data import TokenFunc
+from pyupgrade._token_helpers import replace_name
+
+
+PY2_TYPE = 'Text'
+PY3_TYPE = 'str'
```
Comment (asottile) [inline pyupgrade/_plugins/typing_text.py:16]: I would inline these, they don't really give much as constants

File: tests/features/typing_text_test.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import pytest
+
+from pyupgrade._data import Settings
+from pyupgrade._main import _fix_plugins
+
+
+@pytest.mark.parametrize(
```
Comment (asottile) [inline tests/features/typing_text_test.py:7]: usually there's a set of noop tests here -- you probably want at least (1) no rewrite for < python 3 (2) unrelated names are not rewritten

**Review by asottile — COMMENTED** [review]
otherwise looks good

**Review by asottile — APPROVED** [review]

**Comment by MarcoGorelli** [comment]
Thank you so much for your review and approval!

I'm amazed by how well-structured your code is, I learned a ton just reading through it

**Pull request merged**